packages/office-ui-fabric-react/src/utilities/decorators/withViewport.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { BaseDecorator } from './BaseDecorator';
 import { findScrollableParent, getRect, getWindow } from '../../Utilities';
+import { EventGroup, Async } from '@uifabric/utilities';
```

Comment:
Combine this with the previous import. And if possible it would be better not to use event group.

Reply:
I need Fabric 6.0 compatibility.